services/QuillLMS/app/models/concept_result.rb

```diff
@@ -87,7 +88,10 @@ def self.bulk_create_from_json(data_hash_array)
   def self.find_or_create_from_old_concept_result(old_concept_result)
     return old_concept_result.concept_result if old_concept_result.concept_result
 
-    create_from_json(old_concept_result.as_json)
+    json = old_concept_result.as_json
```

Comment:
I think `data_hash` In lines 91-92, instead of `json` for the variable name would be more clear.

Reply:
That seems like a reasonable change.  I'll make the update.